src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -47,6 +47,10 @@ namespace System.Text.Json
 
         private bool IsLastSpan => _isFinalBlock && (_isSingleSegment || _isLastSegment);
 
+        internal ReadOnlySequence<byte> OriginalSequence => _sequence;
+
+        internal ReadOnlySpan<byte> OriginalSpan => _sequence.IsEmpty ? _buffer : default;
```

Comment:
Do we need the `_sequence.IsEmpty` check?

Reply:
Since the intention is to have the property be "what was passed to the ctor", yes.  Otherwise it reports the span belonging to the current segment.

Reply:
Sure, except all current usages have this under the `OriginalSequence.IsEmpty` check anyway (unless I missed some). Maybe we can expose this differently to avoid the extra checks (internally that is).